### DISCUSSION TITLE
Is SESSION POLICY not supported ?

### DISCUSSION THREAD

**Discussion by tamathew** [Snowflake-Labs/terraform-provider-snowflake#1737] [Q&A]
I'm looking for session policy creation to timeout idle sessions in snowflake. I could not see any resource for that.

**Comment by sfc-gh-ngaberel** [comment]
Hi @tamathew, you're right, session policies aren't supported by this provider yet. If that's a feature you care about I'd suggest creating an issue so we can track it.
The [provider's registry page](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/latest) should always have the latest docs so if you can't find a resource in there then it's not supported.
